src/Common/tests/System/Net/Prerequisites/Servers/CoreFxNetCloudService/WebServer/WebSocket/EchoWebSocket.ashx.cs

```diff
@@ -15,12 +15,15 @@ namespace WebServer
     public class EchoWebSocket : IHttpHandler
     {
         private const int MaxBufferSize = 128 * 1024;
+        private bool _replyWithPartialMessages;
```

Comment:
And even if .NET initializes private bool fields to false, it is better to be explicit about it.